StudCity.Frontend/src/components/Stepper/StepperStrip.jsx

```diff
@@ -7,11 +7,10 @@ const StepperStrip = ({ activeStep }) => {
 			{Array.from({ length: 5 }, (e, index) => (
 				<p
 					key={index}
-					className={` ${styles} ${
-						activeStep === index + 1
-							? "bg-primaryAuthentication w-12"
-							: "bg-stone-200"
-					}`}
+					className={`${styles} ${activeStep === index + 1
```

Comment:
I would suggest to put ternary logic outside of jsx, or just this piece 'activeStep === index + 1' because it's a bad practice to do such things inside className